### ISSUE TITLE
Statistics not working on Internet Explorer 11

### ISSUE THREAD

**Issue by luis100** [keeps/roda#688]
To start, it gives the following error:
```
if (statistics && Array.from(mutation.addedNodes).indexOf(     
LINHA 407 SCRIPT 438: O objeto não suporta a propriedade nem o método 'from'
```

**Comment by luis100** [comment]
http://stackoverflow.com/a/36810954/1987075

**Comment by luis100** [comment]
Fixed on a0b225d592604cc74999f1e36102b73b384fcd35
